prombench/manifests/cluster-infra/7a_commentmonitor_configmap_noparse.yaml

```diff
@@ -26,6 +26,12 @@ data:
       comment_template: |
         Benchmark cancel is in progress.
 
+    - event_type: noop
+      regex_string: (?mi)^/prombench\s*$
+      comment_template: |
+        Please add the version number to compare against.
```

Comment:
```suggestion
        Please add the branch to compare against.
```

Reply:
why should this be the branch?

Reply:
aah yeah you are right is it using the docker image.